templates/terraform/modules/environment/variables.tf

```diff
@@ -156,6 +151,17 @@ variable "sendgrid_api_key_secret_name" {
   type        = string
 }
 
+variable "sendgrid_zone_name" {
+  description = "Route53 zone to create CNAME records for sendgrid authorization"
+  type        = string
+}
+
+variable "sendgrid_domain_prefix" {
+  description = "Prefix for mailing domain used by sendgrid. This will be concatenated with the zone name"
+  type        = string
+  default     = "mail."
```

Comment:
we wanted to remove the prefix last time I believe, and just have it using the root domain

Reply:
Great! The reason I added it here is because I wanted to be able to set it to blank because I just ran into this with helix. 😉

Reply:
I'll leave this here but change the default to blank.